ARCHITECTURE.md

```diff
@@ -68,16 +68,19 @@ the server understands ("SolidDatasets"), and those that do not ("Files").
 The main difference between the two is that, by virtue of the server
 understanding the structure of the former, we can do partial updates (i.e.
 PATCH requests) and retrieve the data in different formats (currently, the
-spec mandates that servers can serve such data as at least Turtle and JSON-LD).
+spec mandates that servers can receive/return such data as at least Turtle and JSON-LD).
 
-Thus, data stored as Turtle or JSON-LD can be fetched as a SolidDataset; the
+Thus, data requested as Turtle or JSON-LD can be fetched as a SolidDataset; the
```

Comment:
I'd say this is still 'wrong'. Basically any data requested as RDF should be fetch-able as a SolidDataset, regardless of the requested serialization (so long as the Solid server happens to support that particular serialization).
In other words, a SolidDataset represents RDF data, and not a particular 'format' of RDF data.